Create security page and document request identity (#431)
restatedev/documentation: docs/develop/java/serving.mdx

</TabItem>
</Tabs>
</details>

> ## Validating request identity

Reviewer: This is missing that you need to pull in the dependency

Reviewer: `dev.restate:sdk-request-identity:VAR::JAVA_SDK_VERSION`

> SDKs are able to validate that incoming requests come from a particular Restate

Reviewer 2: Simpler 😛

```suggestion
SDKs can validate that incoming requests come from a particular Restate
```
